packages/framework-provider-local/src/library/read-model-adapter.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  BoosterConfig,
+  Filter,
+  Logger,
+  ReadModelEnvelope,
+  ReadModelInterface,
+  UUID,
+} from '@boostercloud/framework-types'
+import { ReadModelRegistry } from '../services/read-model-registry'
+
+export async function rawReadModelEventsToEnvelopes(
+  config: BoosterConfig,
+  logger: Logger,
+  rawEvents: Array<unknown>
+): Promise<Array<ReadModelEnvelope>> {
+  return rawEvents.map((event) => event as ReadModelEnvelope)
```

Comment:
If you are not doing any structure conversion, you can simply do:
```suggestion
  return rawEvents as Array<ReadModelEnvelope>
```

Reply:
![](https://media1.tenor.com/images/ebc3fc587c1eae67cb85325e293b987e/tenor.gif?itemid=4268906)

Reply:
Done here https://github.com/boostercloud/booster/pull/473/commits/20070ab9cbf9f4f22d6a72af2357c85bdc86fcd8